pretix_sepadebit/payment.py

```diff
@@ -142,6 +173,13 @@ def payment_is_valid_session(self, request):
             request.session.get('payment_sepa_bic', '') != ''
         )
 
+    def settings_form_clean(self, cleaned_data):
+        super().settings_form_clean(cleaned_data)
+
+        if cleaned_data['payment_sepadebit_earliest_due_date']:
+            if not (cleaned_data['payment_sepadebit_mail_payment_reminder_subject'] and cleaned_data['payment_sepadebit_mail_payment_reminder_text']):
+                raise ValidationError(_("Due date reminder email fields aren't optional if earliest due date feature is used."))
```

Comment:
Let's avoid a double-negative for clarity: 

```suggestion
                raise ValidationError(_("Due date reminder email fields are required if earliest due date feature is used."))
```